src/test/java/com/example/Trello/controllers/BoardControllerTest.java

```diff
@@ -77,17 +77,4 @@ void should_returnBoard_when_boardUpdated() {
         //then
         assertThat(boardEntity).isEqualTo(BOARD_ENTITY);
     }
-
-    @Test
-    void should_returnSuccessfulMessage_when_boardDeleted() {
-        //given
-        willDoNothing().given(boardService).deleteBoard(1);
-
-        //when
-        String message = boardController.deleteBoard(1);
-
-        //then
-        String expectedMessage = "Board has been deleted";
-        assertThat(message).isEqualTo(expectedMessage);
```

Comment:
leave these tests. What you need to do is to check that `boardService .deleteBoard()` is executed with the correct parameter. You can validate it using `then`.
https://www.baeldung.com/bdd-mockito#mockito-vs-bddmockito